src/dfmpreview/src/DfmProcess.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace Dfm_test
+{
+    using Microsoft.DocAsCode.Dfm;
+    using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.Plugins;
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+
+                while (true)
+                {
+                    string num_str = Console.ReadLine();
+                    //Console.WriteLine(tmp + '\n');
+                    int num_of_row = System.Convert.ToInt32(num_str);
```

Comment:
Yes ,fixed

---
In reply to: [73456853](https://github.com/dotnet/docfx/pull/558#discussion_r73456853) [](ancestors = 73456853)